Makefile

```diff
@@ -35,6 +35,18 @@ PY_MAJOR = $(shell python -c 'import sys; print(sys.version[:3])')
 help:
 	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
 
+.PHONY: virtualenv
+virtualenv: ## create virtual environment typically used for development purposes
+	virtualenv env --setuptools --prompt='(twindb_backup)'
+
+.PHONY: bootstrap
+bootstrap: ## bootstrap the development environment
+	pip install -U "setuptools==32.3.1"
```

Comment:
why versions are fixed? can it be a problem in environments with different versions?